notebooks/5094_harmonisation.py

```diff
@@ -217,7 +218,10 @@
     user_overrides_gridding = user_overrides_gridding[user_overrides_gridding != "nan"]
 
 if model.startswith("REMIND"):
-    # advised on 5 June 2025 by Elmar
+    # READING form csv file located in "./data/interim/harmonisation_overrides"
+
+    file = DATA_ROOT / "interim/harmonisation_overrides/harmonisation-methods_gridding_REMIND.csv"
```

Comment:
```suggestion
    file_overrides = DATA_ROOT / "interim/harmonisation_overrides/harmonisation-methods_gridding_REMIND.csv"
```

Slightly clearer name